docs/getting-started-on-bdp/private-managed-cloud/setup-guide-azure/index.md

```diff
@@ -78,6 +78,19 @@ Create a custom role and assign it the “Snowplow BDP Enterprise Deployment”
 ### Determine if Snowplow requires a specific VPC (optional)
 If you require Snowplow to be deployed into a specific VPC CIDR range, this should be provided at the same time as credentials. We need a /16 provided for the VPC so that we can create /20 subnets (note: [VPC peering and using a custom VPC is an additional bolt-on](https://snowplow.io/snowplow-behavioral-data-platform-product-description/#h-vpc-peering-aws-gcp))
 
+### Check subscription resource availablity
+You must check that the following infrastructure is available in the subscription before deplyoment starts. If the required instances are unavailable, it may result in delays to getting started. 
+
+Snowplow requires the below instances and databases. We have provided the Azure CLI commands that can be used to check their availability. Please note that they are required in all availability zones in the region for redundancy. 
+
+Instances DSv3, DSv4 and DSv5 Series:
```

Comment:
The order is important here.  We want the v5 series as they are better than v4 which is better than v3.  Thats better in performance / cost.  Can we order them the other way round ?

Lower case `Series` to `series` ?